tests/utils_tests/test_http.py

```diff
@@ -223,6 +223,21 @@ def test_parsing_rfc850(self):
         parsed = http.parse_http_date('Sunday, 06-Nov-94 08:49:37 GMT')
         self.assertEqual(datetime.utcfromtimestamp(parsed), datetime(1994, 11, 6, 8, 49, 37))
 
+    def test_parsing_rfc850_year(self):
+        dates = (
+            ('Monday, 15-Aug-05 08:49:37 GMT', datetime(2005, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-17 08:49:37 GMT', datetime(2017, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-49 08:49:37 GMT', datetime(2049, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-51 08:49:37 GMT', datetime(2051, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-70 00:00:00 GMT', datetime(1970, 8, 15, 0, 00, 00)),
+            ('Monday, 15-Aug-71 08:49:37 GMT', datetime(1971, 8, 15, 8, 49, 37)),
+            ('Monday, 15-Aug-95 08:49:37 GMT', datetime(1995, 8, 15, 8, 49, 37)),
```

Comment:
Updated, only two tests for two different branches are left

Reply:
I don't think the number of tests is an issue, but since the check depends on a variable, the test should also reflect this, or it'll break at some point.